diff --git i/cmake/modules/FindZephyr-sdk.cmake w/cmake/modules/FindZephyr-sdk.cmake
index 90bbed01..27892f9e 100644
--- i/cmake/modules/FindZephyr-sdk.cmake
+++ w/cmake/modules/FindZephyr-sdk.cmake
@@ -54,7 +54,7 @@ endif()
 # 1) Zephyr specified as toolchain (ZEPHYR_SDK_INSTALL_DIR still used if defined)
 # 2) No toolchain specified == Default to Zephyr toolchain
 # Until we completely deprecate it
-if(("zephyr" STREQUAL ${ZEPHYR_TOOLCHAIN_VARIANT}) OR
+if(("zephyr" STREQUAL "${ZEPHYR_TOOLCHAIN_VARIANT}") OR
    (NOT DEFINED ZEPHYR_TOOLCHAIN_VARIANT) OR
    (DEFINED ZEPHYR_SDK_INSTALL_DIR) OR
    (Zephyr-sdk_FIND_REQUIRED))
